### PR TITLE
[ADF-4576]Add new tests for application cloud component

### PR DESCRIPTION
**Pull request by cristinaj**
**Please check if the PR fulfills these requirements**

> - [ ] The commit message follows our [guidelines](https://github.com/Alfresco/alfresco-ng2-components/wiki/Commit-format)
> - [ ] Tests for the changes have been added (for bug fixes / features)
> - [ ] Docs have been added / updated (for bug fixes / features)

**What kind of change does this PR introduce?** (check one with "x")

> - [ ] Bugfix
> - [ ] Feature
> - [ ] Code style update (formatting, local variables)
> - [ ] Refactoring (no functional changes, no api changes)
> - [ ] Build related changes
> - [ ] Documentation
> - [ ] Other... Please describe:

**What is the current behaviour?** (You can also link to an open issue here)

**What is the new behaviour?**

**Does this PR introduce a breaking change?** (check one with "x")

> - [ ] Yes
> - [ ] No

If this PR contains a breaking change, please describe the impact and migration path for existing applications: ...

**Other information**:
https://issues.alfresco.com/jira/browse/ADF-4648

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -9,55 +9,55 @@
       "resolved": "https://registry.npmjs.org/@alfresco/adf-content-services/-/adf-content-services-3.2.1.tgz",
       "integrity": "sha512-dljtJUnvlA2OC1g9u8TyQEnoHqTEVl94AH6S7dcpWiACLCSMiwd5Zkdpe/p/5FWxPY4on53iIR31KJmyYXfoYQ==",
       "requires": {
-        "tslib": "^1.9.0"
+        "tslib": "1.9.3"
```
Comment (gmandakini) [inline package-lock.json]: why are we changing the package lock file?

File: demo-shell/src/app/components/file-view/file-view.component.html [hunk 2]
```diff
@@ -10,7 +10,7 @@
             <adf-info-drawer-tab label="APP.INFO_DRAWER.PROPERTIES">
                 <adf-content-metadata-card *ngIf="isPreset" [node]="node"
                                            [multi]="multi"
-                                           [preset]="customPreset"
+                                           [preset]="'cris'"
```
Comment (gmandakini) [inline demo-shell/src/app/components/file-view/file-view.component.html]: is this change needed?

**Comment by eromano** [comment]
I merge this PR also if there is one test that is failing but not related to this PR.
